policyengine-client/src/countries/uk/uk.jsx

```diff
@@ -17,32 +17,32 @@ const childNamer = {
 }
 
 function validatePolicy(policy, defaultPolicy) {
-    if(defaultPolicy) {
-        for(let parameter in policy) {
+    if (defaultPolicy) {
+        for (let parameter in policy) {
             policy[parameter].defaultValue = defaultPolicy[parameter].value;
         }
     }
-    if(policy.higher_threshold.value === policy.add_threshold.value) {
+    if (policy.higher_threshold.value === policy.add_threshold.value) {
         policy.higher_threshold.error = "The higher rate threshold must be different than the additional rate threshold.";
         policy.add_threshold.error = "The additional rate threshold must be different than the higher rate threshold.";
-        return {policy: policy, policyValid: false};
+        return { policy: policy, policyValid: false };
     } else {
         policy.higher_threshold.error = null;
         policy.add_threshold.error = null;
     }
-    return {policy: policy, policyValid: true};
+    return { policy: policy, policyValid: true };
 }
 
 export class UK extends Country {
     constructor() {
         super();
         this.baseApiUrl = !this.useLocalServer ?
-                            (
-                                this.usePolicyEngineOrgServer ?
-                                    "https://policyengine.org/" :
-                                    `${window.location.protocol}//${window.location.hostname}`
-                            ) :
-                            `http://localhost:5000`;
+            (
+                this.usePolicyEngineOrgServer ?
+                    "https://policyengine.org/" :
+                    `${window.location.protocol}//${window.location.hostname}`
+            ) :
+            `http://127.0.0.1:5000`;
```

Comment:
Does this address work on other machines too, or should I change it back to localhost?

Reply:
Just tested - works for me, I think it's OK to change it